common/unicode/utf8-utils.cpp

```diff
@@ -991,164 +992,165 @@ int simplify_character(int c) {
   }
 }
 
-const int _s_1__[] = {97, 0};
```

Comment:
TODO may be remove these changes? `const int []` is trivially destructible type, so we can use it in runtime-light